sql/hive/benchmarks/OrcReadBenchmark-jdk21-results.txt

```diff
@@ -2,221 +2,221 @@
-Native ORC Vectorized                                     603            629          35          1.7         574.7       6.0X
+Hive built-in ORC                                        1793           1818          36          0.6        1710.3       1.0X
+Native ORC MR                                            2053           2069          22          0.5        1958.2       0.9X
+Native ORC Vectorized                                     539            569          37          1.9         513.7       3.3X
 
-OpenJDK 64-Bit Server VM 21.0.1+12-LTS on Linux 5.15.0-1053-azure
+OpenJDK 64-Bit Server VM 21.0.2+13-LTS on Linux 6.5.0-1016-azure
 AMD EPYC 7763 64-Core Processor
 Nested Struct Scan with 30 Elements, 10 Fields:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------------
-Hive built-in ORC                                        9213           9334         171          0.1        8786.1       1.0X
-Native ORC MR                                            4514           4529          20          0.2        4305.0       2.0X
-Native ORC Vectorized                                    1521           1550          41          0.7        1450.8       6.1X
+Hive built-in ORC                                        4477           4533          78          0.2        4270.0       1.0X
+Native ORC MR                                            4467           4484          25          0.2        4259.6       1.0X
+Native ORC Vectorized                                    1470           1490          29          0.7        1402.0       3.0X
 
-OpenJDK 64-Bit Server VM 21.0.1+12-LTS on Linux 5.15.0-1053-azure
+OpenJDK 64-Bit Server VM 21.0.2+13-LTS on Linux 6.5.0-1016-azure
 AMD EPYC 7763 64-Core Processor
 Nested Struct Scan with 10 Elements, 30 Fields:  Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------------
-Hive built-in ORC                                        9739           9740           2          0.1        9287.4       1.0X
-Native ORC MR                                            5553           5613          84          0.2        5296.0       1.8X
-Native ORC Vectorized                                    2090           2108          26          0.5        1993.1       4.7X
+Hive built-in ORC                                        4417           4430          19          0.2        4212.1       1.0X
+Native ORC MR                                            5039           5069          42          0.2        4805.9       0.9X
+Native ORC Vectorized                                    1726           1737          16          0.6        1645.8       2.6X
```

Comment:
The new ratio shows that all performance improved in nested struct scan. `Hive built-in ORC` is the most improved one.

Reply:
Impressive! 

Will the related PR be marked release highlight? And does it mean that SPARK-44677 should be planned later?

Reply:
It's too early to say any improvement. I hope there is no regression until Apache Spark 4.0.0 release. :) 

For SPARK-44677, I believe SPARK-44444 would be the precondition. For now, we didn't make any official agreement yet.

Reply:
Thanks for the information. @dongjoon-hyun